Sources/MintKit/Mint.swift

```diff
@@ -52,20 +52,48 @@ public struct Mint {
         return metadata.packages.first(where: { $0.key.lowercased().contains(name.lowercased()) })?.key
     }
 
+    func getGlobalInstalledPackages() -> [String: String] {
+        guard installationPath.exists,
+            let packages = try? installationPath.children() else {
+                print("No mint packages installed to global")
+                return [:]
+        }
+
+        return packages.reduce(into: [:]) { result, package in
+            guard let installStatus = try? InstallStatus(path: package, mintPackagesPath: path),
+                case .mint = installStatus.status,
+                let symlink = try? package.symlinkDestination() else {
+                    return
+            }
+
+            var componets = symlink.components
+            componets.removeLast()
+            let version = componets.removeLast()
+            componets.removeLast()
+            let packageName = String(componets.removeLast().split(separator: "_").last!)
+            result[packageName] = version
+        }
+    }
+
     @discardableResult
     public func listPackages() throws -> [String: [String]] {
         guard packagesPath.exists else {
             print("No mint packages installed")
             return [:]
         }
 
+        let globalInstalledPackages: [String: String] = getGlobalInstalledPackages()
+
         var versionsByPackage: [String: [String]] = [:]
         let packages: [String] = try packagesPath.children().filter { $0.isDirectory }.map { packagePath in
             let versions = try (packagePath + "build").children().sorted().map { $0.lastComponent }
             let packageName = String(packagePath.lastComponent.split(separator: "_").last!)
             var package = "  \(packageName)"
             for version in versions {
                 package += "\n    - \(version)"
+                if globalInstalledPackages[packageName] == version {
+                    package += " *"
```

Comment:
Maybe you could add something about this to the list command description https://github.com/yonaskolb/Mint/blob/master/Sources/MintKit/Commands/ListCommand.swift
`Globally installed packages are marked with *`